Source/WebCore/inspector/agents/InspectorPageAgent.cpp

```diff
@@ -753,7 +760,8 @@ Inspector::Protocol::ErrorStringOr<void> InspectorPageAgent::setCookie(Ref<JSON:
         auto* page = document->page();
         if (!page)
             continue;
-        page->cookieJar().setRawCookie(*document, cookie.value());
+        auto isPartitioned = shouldPartition.value_or(false) ? IsPartitionedCookie::Yes : IsPartitionedCookie::No;
```

Comment:
please move this outside of the `for` since it doesn't rely on the `frame` at all